Mechanisms/Null/mechanism.H

```diff
@@ -13,8 +13,13 @@
 /* Species */
 #define XO_ID 0
 
-#define NUM_ELEMENTS 1
+#ifdef USE_MANIFOLD_EOS
```

Comment:
I did that initially but I think this makes more sense. This is the only change required, so it avoids a lot of code duplication. Because the Null mechanism doesn't get updated automatically when changes are made in ceptr, this reduces the number of manual changes that must be made.